changes/bug33375

```diff
@@ -0,0 +1,6 @@
+  o Minor bugfixes (coding best practices checks):
+    - Stop advertising an IPv6 exit policy when DNS is broken for IPv6 by
+      marking the relay descriptor dirty for 24 hours, not advertising an
+      IPv6 exit policy with a dirty descriptor, and then resetting the
+      IPv6 DNS broken count every 24 hours. Fixes bug 33375; bugfix on
```

Comment:
Can you rephrase the changes file so it's clearer?

It helps to have shorter sentences. Start with a sentence that summarises the change.

You don't need to describe the change in so much detail. Something like "disable the IPv6 exit policy for 24 hours" should be enough.